pkg/tekton/metapipeline/clientfactory.go

```diff
@@ -75,6 +82,11 @@ func NewMetaPipelineClientWithClientsAndNamespace(jxClient versioned.Interface,
 		tektonClient:     tektonClient,
 		kubeClient:       kubeClient,
 		ns:               ns,
+		git:              gitter,
+		stdin:            stdin,
+		stdout:           stdout,
+		stderr:           stderr,
+		teamSettings:     teamSettings,
```

Comment:
I might be mistaken, but do we really need _teamSettings_ as an instance variable?